directories/local-conformance-2_0.v2.json

```diff
@@ -42,7 +43,16 @@
 			"publisher": "FINOS",
 			"icons": [{
 				"src": "http://localhost:3001/finos-icon-256.png"
-			}]
+			}],
```

Comment:
I added the interop element below this so it needed a comma... The listener declared in the interop section is for the basic tests.

Technically, undeclared listeners (dynamic registration) are an optional feature of FDC3 (MAY) so they all need to be declared in the tests. It hasn't caused a fail for anyone as yet, but could and it would be an invalid fail (although its not 100% clear in the spec due to use of an AppD being a SHOULD currently).

Reference: https://fdc3.finos.org/docs/api/spec#register-an-intent-handler